modules/core/src/v2/coins/cspr.ts

```diff
@@ -201,4 +222,67 @@ export class Cspr extends BaseCoin {
       .call(this)
       .asCallback(callback);
   }
+
+  /**
+   * Explain a Casper transaction from Raw Tx
+   *
+   * @param {ExplainTransactionOptions} params given explain transaction params
+   * @param {String} params.txHex raw transaction
+   * @param {String} params.halfSigned.txHex raw half signed transaction
+   * @param {TransactionFee} fee fee information
+   * @param {Function} callback
+   * @returns Bluebird<TransactionExplanation>
+   */
+  explainTransaction(
+    params: ExplainTransactionOptions,
+    callback?: NodeCallback<TransactionExplanation>
+  ): Bluebird<TransactionExplanation> {
+    const self = this;
+    return co<TransactionExplanation>(function*() {
+      const txHex = params.txHex || (params.halfSigned && params.halfSigned.txHex);
+      if (!txHex || !params.fee) {
+        throw new Error('missing explain tx parameters');
+      }
+      const txBuilder = accountLib.getBuilder(self.getChain()).from(txHex);
+
+      const tx: any = yield txBuilder.build();
+      if (!tx) {
+        throw new InvalidTransactionError('Error while trying to build transaction');
+      }
+      const id = Buffer.from(tx.casperTx.hash).toString('hex');
+      const transferId = accountLib.Cspr.Utils.getTransferId(tx.casperTx.session);
+      const amount = accountLib.Cspr.Utils.getTransferAmount(tx.casperTx.session);
+      const toAddress = accountLib.Cspr.Utils.getTransferDestinationAddress(tx._deploy.session);
+      const outputs: TransactionOutput[] = [];
+      // TODO(https://bitgoinc.atlassian.net/browse/STLX-677): define outputs for different types of tx
+      if (tx.type === accountLib.BaseCoin.TransactionType.Send) {
+        outputs.push({
+          address: toAddress,
+          amount,
+          coin: self.getChain(),
```

Comment:
Seems this is always set to the same value - is there some other reason this is needed?